src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -1672,10 +1728,12 @@ PlanRouterQuery(Query *originalQuery, RelationRestrictionContext *restrictionCon
 	uint64 shardId = INVALID_SHARD_ID;
 	CmdType commandType = originalQuery->commandType;
 	bool isMultiShardModifyQuery = false;
+	bool isMultipleTableModifyQuery = false;
 
 	*placementList = NIL;
 	prunedRelationShardList = TargetShardIntervalsForQuery(originalQuery,
```

Comment:
This looks really hard to read. Can you add a newline before `TargetShardIntervalsForQuery `